docs/development.rst

```diff
@@ -29,19 +29,19 @@ Some guidance for contributors:
   - if you need to fix something after commit/push:
 
     - if there are ongoing reviews: do a fixup commit you can
-      merge into the bad commit later.
+      squash into the bad commit later.
     - if there are no ongoing reviews or you did not push the
-      bad commit yet: edit the commit to include your fix or
+      bad commit yet: amend the commit to include your fix or
       merge the fixup commit before pushing.
   - have a nice, clear, typo-free commit comment
   - if you fixed an issue, refer to it in your commit comment
   - follow the style guide (see below)
 
 - if you write new code, please add tests and docs for it
 
-- run the tests, fix anything that comes up
+- run the tests, fix any errors that come up
```

Comment:
Well, "anything" sounds quite strange here. There are "things" (such as the summary) coming up in any case, even if there are no errors. Maybe "fix any issues" instead? I know that this is just nitpicking. But then again, the whole patch is just minor tweaks...